controllers/metal3remediation_controller_test.go

```diff
@@ -35,23 +35,51 @@ import (
 	corev1 "k8s.io/api/core/v1"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/client/fake"
 )
 
+var (
+	ownerMachineErrorMsg    = "metal3Remediation's owner Machine could not be retrieved"
+	ownerMachineNotFoundMsg = "metal3Remediation's owner Machine not set"
+	metal3machineErrorMsg   = "metal3machine not found"
```

Comment:
What do you think about renaming the variables here so they are closer to the messages? Something like this maybe?
```suggestion
	ownerMachineErrorMsg    = "metal3Remediation's owner Machine could not be retrieved"
	ownerMachineNotSetMsg = "metal3Remediation's owner Machine not set"
	metal3machineNotFoundMsg   = "metal3machine not found"
```